tests/merch.test.ts

```diff
@@ -564,4 +569,81 @@ describe('merch order pickup events', () => {
     expect(persistedPickupEvent.orders).toHaveLength(1);
     expect(persistedPickupEvent.orders[0]).toStrictEqual(persistedOrder);
   });
+
+  test('placing an order with a pickup event that has reached the order limit fails', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake({
```

Comment:
Let's add some happy-path tests as well, like 'placing an order with a pickup event that has not reached its capacity succeeds', or 'updating a pickup event's order limit to a threshold above the current order count succeeds'

Reply:
Resolved in `6c62606` with some issues. See comments.